pkg/workflows/builder.go

```diff
@@ -1,35 +1,184 @@
 package workflows
 
-// The real implementations will come in a follow-up PR.
-// these stubs allow the code from the generators to compile.
-// A holistic view can be seen at https://github.com/smartcontractkit/chainlink-common/pull/695
+import (
+	"fmt"
+	"strconv"
+	"strings"
 
-type WorkflowSpecFactory struct{}
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+)
+
+// 1. Cap defines JSON schema for inputs and outputs of a cap.
+// Trigger: triggerOutputType := workflowBuilder.addTrigger(DataStreamsTrigger.ModifiedConfig{})
+// Adds metadata to the builder. Returns output type.
+// 2. Consensus: consensusOutputType := workflowBuilder.addConsensus(ConsensusConfig{
+// 	Inputs: triggerOutputType,
+// })
```

Comment:
Detached comments don't render will in docs. Should this be a package level comment at the top of the file?

Reply:
It is an old comment from when @DeividasK and I were exploring that never got removed, so I'll remove it.